terraform/modules/network/resources.tf

```diff
@@ -26,56 +26,63 @@ resource "aws_security_group" "default" {
       from_port   = 0
       to_port     = 0
       protocol    = "-1"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
```

Comment:
This allows all ports to the addresses in the cidr_block. This makes all next ingress rules irrelevant.